frontend/src/containers/Home.scss

```diff
@@ -0,0 +1,14 @@
+.usa-search__submit-text {
+  vertical-align: text-top !important;
+}
+
+.cards ul {
+  display: flex;
+  flex-wrap: wrap;
```

Comment:
Did you check the USWDS flex?

https://designsystem.digital.gov/utilities/flex/

Reply:
Yes, but the appearance this way better matched the Figma design.